pello/pello_greeting.py

```diff
@@ -1,3 +1,3 @@
 def greeting():
     print("Hello World!")
-
+    return "Hello World!"
```

Comment:
Does this not break the entry point?

Reply:
No, it does not. It prints it on stdout so I removed the print above to not print it twice.

Reply:
What code prints it now?

Reply:
BTW we have debugged this together and the text was printed to stderr, because the entrypoint basically boiled down to `sys.exit("Hello World!")`.